src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -22,45 +20,34 @@ public class Ditto
         /// <typeparam name="TObjectType">The type of the object being converted.</typeparam>
-        /// <typeparam name="TResolverType">The type of the value resolver.</typeparam>
-        public static void RegisterValueResolver<TObjectType, TResolverType>()
-            where TResolverType : DittoValueResolver
+        /// <typeparam name="TProcessorType">The type of the processor type.</typeparam>
+        public static void RegisterProcessor<TObjectType, TProcessorType>()
+            where TProcessorType : DittoProcessor
         {
-            DittoValueResolverRegistry.Instance.RegisterResolver<TObjectType, TResolverType>();
+            DittoProcessorRegistry.Instance.RegisterProcessor<TObjectType, TProcessorType>();
         }
 
         /// <summary>
         /// Registers a global value resolver attribute.
         /// </summary>
         /// <typeparam name="TObjectType">The type of the object being converted.</typeparam>
-        /// <typeparam name="TResolverAttributeType">The type of the value resolver attribute.</typeparam>
-        public static void RegisterValueResolverAttribute<TObjectType, TResolverAttributeType>()
-            where TResolverAttributeType : DittoValueResolverAttribute, new()
+        /// <typeparam name="TProcessorAttributeType">The type of the processor attribute type.</typeparam>
+        public static void RegisterProcessorAttribute<TObjectType, TProcessorAttributeType>()
+            where TProcessorAttributeType : DittoProcessorAttribute, new()
         {
-            DittoValueResolverRegistry.Instance.RegisterResolverAttribute<TObjectType, TResolverAttributeType>();
+            DittoProcessorRegistry.Instance.RegisterProcessorAttribute<TObjectType, TProcessorAttributeType>();
         }
 
         /// <summary>
         /// Registers a global value resolver attribute.
         /// </summary>
         /// <typeparam name="TObjectType">The type of the object being converted.</typeparam>
-        /// <typeparam name="TResolverAttributeType">The type of the value resolver attribute.</typeparam>
+        /// <typeparam name="TProcessorAttributeType">The type of the processor attribute type.</typeparam>
         /// <param name="instance">An instance of the value resolver attribute to use.</param>
-        public static void RegisterValueResolverAttribute<TObjectType, TResolverAttributeType>(TResolverAttributeType instance)
-            where TResolverAttributeType : DittoValueResolverAttribute
-        {
-            DittoValueResolverRegistry.Instance.RegisterResolverAttribute<TObjectType, TResolverAttributeType>(instance);
-        }
-
-        /// <summary>
-        /// Registers a global type converter.
-        /// </summary>
-        /// <typeparam name="TObjectType">The type of the object being converted.</typeparam>
-        /// <typeparam name="TConverterType">The type of the converter.</typeparam>
-        public static void RegisterTypeConverter<TObjectType, TConverterType>()
```

Comment:
I'm wondering whether we should keep hold of the `RegisterTypeConverter` helper in facade.
(I found it simpler syntax - also that we do make a call to Umbraco's `TryConvertTo` in `.GetConvertedValue` :wink:)